jekyll/_cci2/deployment-integrations.md

```diff
@@ -109,24 +109,26 @@ To deploy to Azure, use a similar job to the above example that uses an appropri
 The built-in Heroku integration through the CircleCI UI is not implemented for CircleCI 2.0. However, it is possible to deploy to Heroku manually by using a script to set up Heroku, adding SSH keys with the `add_ssh_keys` option and configuring a workflow. 
 
 1. Create a script to set up Heroku similar to this example `setup-heroku.sh` file in the `.circleci` folder:
-     ```
-     #!/bin/bash
-     wget https://cli-assets.heroku.com/branches/stable/heroku-linux-amd64.tar.gz
-     sudo mkdir -p /usr/local/lib /usr/local/bin
-     sudo tar -xvzf heroku-linux-amd64.tar.gz -C /usr/local/lib
-     sudo ln -s /usr/local/lib/heroku/bin/heroku /usr/local/bin/heroku
-
-     cat > ~/.netrc << EOF
-     machine api.heroku.com
-       login $HEROKU_LOGIN
-       password $HEROKU_API_KEY
-     EOF
-
-     cat >> ~/.ssh/config << EOF
-     VerifyHostKeyDNS yes
-     StrictHostKeyChecking no
-     EOF
-     ```
+
```

Comment:
I think it is correct not to have the tabs in this yaml, but it does break the step numbering, so we should not use steps here.